src/components/ArticleView/CommentEntry.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { postCommentByArticleId } from '../../../api';
+
+const CommentEntry = ({ article_id, onCommentPosted }) => {
+    const [userComment, setUserComment] = useState('');
+
+    function handlePostComment() {
+        postCommentByArticleId(article_id, {
+            username: 'cooljmessy',
+            body: userComment,
+        })
+        .then(() => {
+            setUserComment('');
+            onCommentPosted();
```

Comment:
I don't think this is the best name for the function, it doesn't really tell me what this does (I had to look at your other component), I guess it's loading your comments?